helm/hpcc/templates/_helpers.tpl

```diff
@@ -139,6 +139,33 @@ Returns the largest number of workers from all the thors
  {{- $maxNumWorkers -}}
 {{- end -}}
 
+{{/*
+Returns true if the given certificate issuer is enabled, otherwist false
+*/}}
+{{- define "hpcc.isIssuerEnabled" -}}
+{{- $certificates := (.root.Values.certificates | default dict) -}}
+{{- $issuers := ($certificates.issuers | default dict) -}}
+{{- $issuer := get $issuers .issuer -}}
+{{- if ($issuer) -}}
```

Comment:
trivial/style: don't think brackets needed/normally used in this type of context